jdk9/plan.sh

```diff
@@ -0,0 +1,81 @@
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_name=jdk9
+pkg_version=9.0.1
+pkg_source=http://download.oracle.com/otn-pub/java/jdk/${pkg_version}+11/jdk-${pkg_version}_linux-x64_bin.tar.gz
+pkg_shasum=2cdaf0ff92d0829b510edd883a4ac8322c02f2fc1beae95d048b6716076bc014
+pkg_filename=jdk-${pkg_version}_linux-x64_bin.tar.gz
+pkg_license=('Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX')
+pkg_description=('Oracle Java Development Kit. This package is made available to you to allow you to run your applications as provided in and subject to the terms of the Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX, found at http://www.oracle.com/technetwork/java/javase/terms/license/index.html')
+pkg_upstream_url=http://www.oracle.com/technetwork/java/javase/overview/index.html
+pkg_deps=(core/glibc core/gcc-libs core/zlib)
+pkg_build_deps=(core/patchelf core/file)
+pkg_bin_dirs=(bin jre/bin)
+pkg_lib_dirs=(lib lib/amd64)
+pkg_include_dirs=(include)
+
+source_dir=$HAB_CACHE_SRC_PATH/${pkg_name}-${pkg_version}
+
+## Refer to habitat/components/plan-build/bin/hab-plan-build.sh for help
+
+# Customomized download_file() to work around the Oracle EULA Cookie-wall
+#  See: http://stackoverflow.com/questions/10268583/downloading-java-jdk-on-linux-via-wget-is-shown-license-page-instead
+download_file() {
```

Comment:
I think we can reduce a lot of the copy/paste here if we source the jdk/jre plans at the top of this file

Reply:
Done!